README.md

```diff
@@ -95,6 +95,7 @@ The generator (cookiecutter) will ask you for some data, you might want to have
 The input variables, with their default values (some auto generated) are:
 
 * `project_name`: The name of the project
+* `project_version`: The version of the project
```

Comment:
```suggestion
```

I would prefer to have a default fixed version `0.1.0` for all new generated projects and let everyone customize ii afterwards if needed.